### PR TITLE
DD4.18 scout: update cli documentation

### PR DESCRIPTION
**Pull request by eunomie**
### Proposed changes

Update `docker scout` cli plugin documentation based on version `0.8.5` for DD 4.18

### CODE REVIEW

**Comment by eunomie** [comment]
I haven't yet looked at it, but I think something is missing to get all the commands listed here:
image

File: _data/scout-cli/docker_scout_compare.yaml [hunk 1]
```diff
@@ -0,0 +1,212 @@
+command: docker scout compare
+short: '[early preview] Compare two images and display differences'
+long: |-
+    The `docker scout compare` command analyzes two images and display a comparison of both.
```
Comment (ChrisChinchilla) [inline _data/scout-cli/docker_scout_compare.yaml:4]:
```suggestion
    The `docker scout compare` command analyzes two images and displays a comparison of both.
```

File: _data/scout-cli/docker_scout_compare.yaml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+    The following artifact types are supported:
+
+    - Images
+    - OCI layout directories
+    - Tarball archives, as created by `docker save`
+
+    The tool analyzes the provided software artifact, and generates a vulnerability report.
+
+    By default, the tool expects an image reference, such as:
```
Comment (ChrisChinchilla) [inline _data/scout-cli/docker_scout_compare.yaml:19]:
```suggestion
    By default, the command expects an image reference, such as:
```

File: _data/scout-cli/docker_scout_compare.yaml [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: only-package-type
+      value_type: stringSlice
+      default_value: '[]'
+      description: |
+        Comma separated list of package types (like apk, deb, rpm, npm, pypi, golang, etc)
```
Comment (ChrisChinchilla) [inline _data/scout-cli/docker_scout_compare.yaml:77]:
```suggestion
        Comma separated list of package types (e.g. apk, deb, rpm, npm, pypi, golang, etc)
```

File: _data/scout-cli/docker_scout_quickview.yaml [hunk 4]
```diff
@@ -0,0 +1,91 @@
+command: docker scout quickview
+short: Quick overview of an image
+long: |-
+    The `docker scout quickview` command displays a quick overview of an image.
+    It displays a summary of the vulnerabilities in the image and the one specifically in the base image.
```
Comment (ChrisChinchilla) [inline _data/scout-cli/docker_scout_quickview.yaml:5]: @eunomie What does "the one specifically" mean? Maybe is it the vulnerabilities in the image that come from the base image?

File: _data/scout-cli/docker_scout_sbom.yaml [hunk 5]
```diff
@@ -0,0 +1,135 @@
+command: docker scout sbom
+short: Generate or display SBOM of an image
+long: |-
+    The `docker scout sbom` command analyzes a software artifact to generate the corresponding Software Bill Of Materials - SBOM.
```
Comment (ChrisChinchilla) [inline _data/scout-cli/docker_scout_sbom.yaml:4]:
```suggestion
    The `docker scout sbom` command analyzes a software artifact to generate the corresponding Software Bill Of Materials (SBOM).
```

**Review by ChrisChinchilla — CHANGES_REQUESTED** [review]
A few small typos and a question @eunomie 

Also some things that I might tweak at a later date for style consistency, but less important than getting this out.

**Comment by ChrisChinchilla** [comment]
@eunomie Looks like menu entries are manual, added them for you @eunomie

File: _data/scout-cli/docker_scout_compare.yaml (hunk 2 above)
Comment (eunomie) [reply _data/scout-cli/docker_scout_compare.yaml:19]: Not sure about this one, or we need to change it in all other commands (this part is just a copy/paste of the others)

File: _data/scout-cli/docker_scout_compare.yaml (hunk 3 above)
Comment (eunomie) [reply _data/scout-cli/docker_scout_compare.yaml:77]: Same, I'm tempted to keep it for now and then have a pass over all the scout docs, so it's consistent across all the commands.

**Review by ChrisChinchilla — APPROVED** [review]

**Pull request merged**